### PR TITLE
[TFLite] Add int8 and int16x8 support for TRANSPOSE_CONV_3D operator

### PR DESCRIPTION
**Pull request by georgeedward2000**
Hello,

This PR adds quantizable int8 and int16x8 implementations of Conv3DTranspose operator.
This PR depends on the [PR that adds support for int8 and int16x8 Conv3D operator](https://github.com/tensorflow/tensorflow/pull/51527) to be merged first.

Cheers,
Eddie

### CODE REVIEW

**Comment by abattery** [comment]
@teijeong @thaink could you review this PR?

**Comment by gbaned** [comment]
@teijeong, @thaink  Can you please review this PR ? Thanks!

**Comment by gbaned** [comment]
@georgeedward2000 Can you please resolve conflicts? Thanks!

**Comment by gbaned** [comment]
@georgeedward2000 Can you please resolve conflicts? Thanks!

**Comment by gbaned** [comment]
@georgeedward2000 Can you please resolve conflicts? Thanks!

**Comment by SaoirseARM** [comment]
> @georgeedward2000 Can you please resolve conflicts? Thanks!

Thanks, The merge conflict has now been resolved.
Best regards,
Saoirse

**Comment by gbaned** [comment]
@teijeong, @thaink Can you please review this PR ? Thanks!

**Comment by abattery** [comment]
@alankelly could you review this PR request?

File: tensorflow/compiler/mlir/lite/ir/tfl_ops.td
Comment (abattery) [inline tensorflow/compiler/mlir/lite/ir/tfl_ops.td:5416]: Apparently, there are more discussions for which the per channel dimension location should be supported in this operator in the long term.

Could you provide the rationale for your choice here?

**Review by abattery — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@georgeedward2000 Can you please check @abattery's comments and keep us posted ? Thank you!

**Comment by gbaned** [comment]
Hi @georgeedward2000 Any update on this PR? Please. Thank you!

**Comment by Tessil** [comment]
@gbaned We're currently waiting for a reply of @abattery on https://github.com/tensorflow/tensorflow/pull/51527#discussion_r1003403774 which is a similar question. Once we know how to proceed with this other PR, we'll update this one if necessary.

**Comment by gbaned** [comment]
Hi @abattery Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @abattery Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @abattery Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @grantjensen Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @grantjensen / @abattery Any update on this PR? Please. Thank you!

**Comment by grantjensen** [comment]
Dependent on https://github.com/tensorflow/tensorflow/pull/51527. Will wait to give final review until after it is merged...

**Comment by keerthanakadiri** [comment]
Hi @Linchenn ,Can you please review this PR? Thank you !

**Comment by keerthanakadiri** [comment]
Hi @georgeedward2000, Can you please resolve the conflicts? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @georgeedward2000, Can you please resolve the conflicts? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @georgeedward2000, Can you please resolve the conflicts? Thank you!

**Pull request closed without merging**
